nautobot/docs/plugins/development.md

```diff
@@ -574,6 +574,12 @@ A `PluginMenuButton` has the following attributes:
 !!! note
     Any buttons associated within a menu item will be shown only if the user has permission to view the link, regardless of what permissions are set on the buttons.
 
+## Modifying Existing NavBar
+
+Plugins can modify the existing navigation bar layout by defining `menu_items` inside of `navigation.py`. Using the key and weight system, a developer can integrate the plugin amongst existing menu tabs, groups, items and buttons and/or create entirely new menus as desired.
+
+More documentation and examples can be found [here](../development/navigation-menu.md)
+
```

Comment:
This section should be integrated into (and perhaps replace entirely) the above `Navigation Menu Items` section.